cmd/packages.pony

```diff
@@ -93,6 +97,7 @@ class val Package is Comparable[Package box]
     match os
     | Linux => fragments.push("linux")
     | Darwin => fragments.push("darwin")
+    | FreeBSD => fragments.push("freebsd")
```

Comment:
We can replace this match using the improved `OS` type definition described below, since it provides the `string()` function

Reply:
Do you want to push those changes onto this branch?

Reply:
Sure, give me a few minutes